### DISCUSSION TITLE
Renaming repo dont update the links on readme href

### DISCUSSION THREAD

**Discussion by Sakretsos** [upptime/upptime#711] [General] [answered]
Hello,

Today i realize an issue on my upptime monitor repo.
When i rename my repo the href links on readme file still use the old repo link (the original one).
Is this a problem from the script or i must do something else in order to get updated?

Thanks,
Ioannis

**Comment by AnandChowdhary** [comment, marked as answer]
You can manually run "Setup CI" after renaming your repository, that should take care of everything! :D
